openfisca_canada/variables/canadian_emergency_relief_benefit.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+# This file defines variables for the modelled legislation.
+# A variable is a property of an Entity such as a Person, a Household…
+# See https://openfisca.org/doc/key-concepts/variables.html
+
+# Import from openfisca-core the common Python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the Entities specifically defined for this tax and benefit system
+from openfisca_canada.entities import *
+
+class is_a_resident_of_canada(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant is a resident of Canada.'
+
+
+class believe_they_can_get_EI(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant believes they can recieve' \
+            'EI Regular or Sickness Benefits'
+
+class quit_their_job_voluntarily(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant quit their job voluntarily'
+
+class has_lost_income_for_half_of_the_application_period(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant has lost their income for at least' \
+            'half of the four week period for which they are applying'
+
+class they_are_over_15_years_of_age(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant is over the age of 15 at time of applying'
+
+class minimum_earnings_amount(Variable): 
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant has earned a minimum of $5,000  '
```

Comment:
For this one, it might be better to have a variable for earning amount - but with the terms in the name, like "annual_earnings_amount" as a person-related variable, then have a parameter for the minimum amount for the emergency relief benefit - in this case, 5000 CAD.

Reply:
Fixed